packages/react-charting/src/components/Legends/Legends.test.tsx

```diff
@@ -100,10 +160,25 @@ describe('AreaChart - basic props', () => {
     expect(legend).toBeDefined();
   });
 
-  it('Should render overflowProps correctly', () => {
-    wrapper = mount(<Legends legends={legends} {...overflowProps} />);
-    const overflowProp = wrapper.getDOMNode().querySelectorAll('[class^="ms-Layer"]');
-    expect(overflowProp).toBeDefined();
+  it('Should mount Overflow Button when not empty', () => {
+    wrapper = mount(<Legends legends={legends} {...overflowProps} overflowText={'OverFlow Items'} />);
+    const overflowBtnText = wrapper.getDOMNode().querySelectorAll('[class^="ms-OverflowSet-overflowButton"]');
+    expect(overflowBtnText).toBeDefined();
+  });
+
+  it('Should not mount Overflow when empty', () => {
+    wrapper = mount(<Legends legends={legends} {...overflowProps} overflowText={'OverFlow Items'} />);
+    const overflowBtn = wrapper.getDOMNode().querySelectorAll('[class^="ms-OverflowSet-overflowButton"]');
+    expect(overflowBtn!.length).toBe(0);
```

Comment:
How did overflow became empty for this case.

Reply:
resolving this as I had an external conversation with Atishay